pyramid_oereb/standard/create_tables.py

```diff
@@ -124,3 +137,85 @@ def create_standard_tables():
                 c2ctemplate_style=options.c2ctemplate_style,
                 sql_file=sql_file
             )
+
+
+def create_theme_tables():
+    parser = optparse.OptionParser(
+        usage='usage: %prog [options]',
+        description='Create all content for the standard database'
+    )
+    parser.add_option(
+        '-c', '--configuration',
+        dest='configuration',
+        metavar='YAML',
+        type='string',
+        help='The absolute path to the configuration yaml file.'
+    )
+    parser.add_option(
+        '-s', '--section',
+        dest='section',
+        metavar='SECTION',
+        type='string',
+        default='pyramid_oereb',
+        help='The section which contains configuration (default is: pyramid_oereb).'
+    )
+    parser.add_option(
+        '-t', '--theme',
+        dest='theme',
+        metavar='CODE',
+        type='string',
+        help='The code of the theme to be created.'
+    )
+    parser.add_option(
+        '-T', '--tables-only',
+        dest='tables_only',
+        action='store_true',
+        default=False,
+        help='Use this flag to skip the creation of the schema.'
+    )
+    parser.add_option(
+        '--sql-file',
+        type='string',
+        help='Generate an SQL file.'
+    )
+    parser.add_option(
+        '--c2ctemplate-style',
+        dest='c2ctemplate_style',
+        action='store_true',
+        default=False,
+        help='Is the yaml file using a c2ctemplate style (starting with vars)'
+    )
+    options, args = parser.parse_args()
+    if not options.configuration:
+        parser.error('No configuration file set.')
+    if not options.theme:
+        parser.error('No theme code specified.')
+
+    if Config.get_config() is None:
+        Config.init(
+            options.configuration,
+            options.section,
+            options.c2ctemplate_style
+        )
+
+    theme_config = None
+    for cfg in Config.get('plrs'):
```

Comment:
Maybe use direct method of Config object here: https://github.com/openoereb/pyramid_oereb/blob/2861ef390bb7eae7a4f42df62086be4ffa373f60/pyramid_oereb/lib/config.py#L815

Reply:
Config object got really handy ;)

Reply:
Thanks for the hint :+1: 
I've applied the method.